dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/RelationshipService.java

```diff
@@ -62,6 +62,8 @@ Page<Relationship> getRelationships(RelationshipOperationParams params, PagePara
   List<Relationship> getRelationships(@Nonnull Set<UID> uids)
       throws ForbiddenException, NotFoundException;
 
+  List<Relationship> getUidsByRelationshipKeys(List<String> relationshipKeyList);
```

Comment:
```suggestion
  List<Relationship> getUidsByRelationshipKeys(List<String> relationshipKeys);
```

Can you please add some javadoc on what that key even is 😅

Reply:
Done